src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -24,7 +24,7 @@ import { AbstractAI, AIOptions, AIStats } from 'src/app/jscaip/AI';
 @Debug.log
 export class LocalGameWrapperComponent extends GameWrapper<string> implements AfterViewInit {
 
-    public static readonly AI_TIMEOUT: number = 1000;
+    public static readonly AI_TIMEOUT: number = 1;
```

Comment:
j'imagine que c'était pour tester mais qu'on veut pas ça en vrai ? :D

Reply:
Ta réponse est donc :
    // TODO FOR REVIEW: euh non 1ms c'est mieux, je crois que ça fait partie de zéro remarques "d'user"
    //  "oh non l'ia joue trop vite, je ne comprend pas quand elle joue"
    // Alors que la voir jouer super vite contre elle même c'est trop bieeeeen
    // Tu veux vraiment que je remette :p ?
Bah je dirais : ok pour que ça soit 1ms, ça me dérange pas je crois ! Au pire on pourrait rendre cela configurable : un paramètre (peut-être caché, que pour les devs & experts) qui permet d'avoir une IA non-ralentie ?